### DISCUSSION TITLE
Can't show TABLE in epub file

### DISCUSSION THREAD

**Discussion by jp311** [sumatrapdfreader/sumatrapdf#3128] [General]
like this picture, left is SumatraPDF, right is calibre Viewer. SumatraPDF can't show Table.
![屏幕截图 2022-11-24 174353](https://user-images.githubusercontent.com/336683/203751385-29f4914c-8007-4390-972c-ecfa5af69658.png)

**Comment by GitHubRulesOK** [comment]
see https://github.com/sumatrapdfreader/sumatrapdf/issues/319 ,
https://github.com/sumatrapdfreader/sumatrapdf/issues/565


https://github.com/sumatrapdfreader/sumatrapdf/issues/849
https://github.com/sumatrapdfreader/sumatrapdf/issues/1441
https://github.com/sumatrapdfreader/sumatrapdf/issues/2272

especially https://github.com/sumatrapdfreader/sumatrapdf/issues/2272#issuecomment-1146881609
